src/pretix/base/validators.py

```diff
@@ -0,0 +1,45 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+class BlacklistValidator:
+
+    blacklist = []
+
+    def __call__(self, value):
+        # Validation logic
+        if value in self.blacklist:
+            message = _("This slug has an invalid value.")
```

Comment:
If you want to change this, see also https://docs.djangoproject.com/en/1.10/ref/forms/validation/#raising-validationerror about how to use parameters here